docs/pages/next_major_development.md

```diff
@@ -20,6 +20,48 @@ The current tasks are listed here: [Integrate new PMD logo #1931](https://github
 
 The new logo is available from the [Logo Project Page](pmd_projectdocs_logo.html).
 
+### Revamped Command Line Interface
+
+PMD now ships with a unified Command Line Interface for both Linux/Unix and Windows. Instead of having a collection of scripts 
+for the different utilities shipped with PMD, a single script `pmd` (`pmd.bat` for Windows) can now launch all
+utilities using subcommands, e.g. `pmd run`, `pmd designer`. All commands and options are thoroughly documented in the help,
+with full color support where available. Moreover, efforts were made to provide consistency in the usage of all PMD utilities.
+
+```
+$ pmd --help    
+Usage: pmd [-hV] [COMMAND]
+  -h, --help      Show this help message and exit.
+  -V, --version   Print version information and exit.
+Commands:
+  analyze, analyse, run  The PMD standard source code analyzer
```

Comment:
I'm not sure we should alias this command. `analyze` is long and has spelling variants, so maybe we should avoid it. `run` is a bit nondescript... Maybe `pmd check` would be appropriate? In Rust you call `cargo check` to check your code

Reply:
:+1: I like `check`.

Reply:
Agreed, changing it!